ui/lib/utils.ts

```diff
@@ -147,9 +154,9 @@ export const convertImage = (image: string) => {
     !prefix.includes("public.ecr.aws") &&
     !prefix.includes("amazonaws.com")
```

Comment:
Sorry to be reacting to things that weren't part of this PR :see_no_evil: 

I think you need to handle this case by counting slashes - if there's 0 slashes then the link is `hub.docker.com/_/image_name` (no r!), if there's 1 slash then the link is `hub.docker.com/r/image_name`, if there's 2 slashes then the first part is a domain and domain checking applies.

Reply:
No worries happy you're looking at it! Didn't realize there were cases without the `/r/` - will change it